static/app/components/performance/waterfall/utils.tsx

```diff
@@ -25,9 +25,9 @@ export const getBackgroundColor = ({
   return theme.background;
 };
 
-export function getHatchPattern(spanBarHatch: SpanBarHatch | undefined) {
-  if (spanBarHatch) {
-    const {primary, alternate} = SPAN_HATCH_TYPE_COLOURS[spanBarHatch];
+export function getHatchPattern(spanBarType: SpanBarType | undefined, theme: Theme) {
+  if (spanBarType) {
+    const {primary, alternate} = getSpanBarColours(theme, spanBarType);
```

Comment:
TypeScript makes a mistake impossible, but it's still a little strange that the order of arguments for `getHatchPattern` and `getSpanBarColours` is type, theme for the former and theme, type for the latter

Reply:
Ah, makes sense! Up to you